modules/standard/Time.chpl

```diff
@@ -573,7 +573,7 @@ enum Day       { sunday=0, monday, tuesday, wednesday, thursday, friday, saturda
     pragma "no doc"
     var chpl_hour, chpl_minute, chpl_second, chpl_microsecond: int;
     pragma "no doc"
-    var chpl_tzinfo: shared TZInfo?;
+    @unstable "tzinfo is unstable; its type may change in the future" : shared TZInfo?;
```

Comment:
I think this is what's causing the failure - looks like you replaced `var chpl_tzinfo` with the unstable keyword.  Though, I think it might be better to just attach it to the functions that have those arguments - putting it on the field will impact any uses in the methods and give warnings that the user wouldn't be able to do anything about.  But they can choose to call different functions